var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -19,15 +19,13 @@ class NeurodamusModel(SimModel):
     # NOTE: Several variants / dependencies come from SimModel
     variant('synapsetool', default=True,  description="Enable SynapseTool reader (for edges)")
     variant('mvdtool',     default=True , description="Enable MVDTool reader (for nodes)")
-    variant('python',      default=False, description="Install neurodamus-python alongside")
     variant('common_mods', default='',    description="Source of common mods. '': no change,"
                                                       " other string: alternate path")
 
-    depends_on('neurodamus-core')
-    depends_on('neurodamus-core@develop', when='@develop')
-    depends_on('neurodamus-core+python', when='+python')
+    depends_on('neurodamus-core', type='build')
+    depends_on('neurodamus-core@develop', type='build', when='@develop')
 
-    depends_on("mpi")
+    depends_on("mpi", type=('build', 'run'))
```

Comment:
mpi is typically link as well.
```suggestion
    depends_on("mpi")
```

MPI is also typically link dependency.

Reply:
In our setup it was problematic, we cant do `spec['mpi'].libs (maybe something to be fixed)
But since we now use neuron with Dynamic MPI, isn’t this the desired way?

Reply:
But for final exe .i.e. special we link with MPI wrappers through NEURON. So I will just say depends_on(‘mpi’).

Reply:
I sincerely had it like that, but `spec['mpi'].libs` failed (at build time, saying it cannot find the libraries). And since we use wrappers, which do the linking for us, I don't see a better alternative.